adhocracy4/exports/views.py

```diff
@@ -67,7 +67,12 @@ def get_base_filename(self):
         return '%s_%s' % ('download',
                           timezone.now().strftime('%Y%m%dT%H%M%S'))
 
+    def strip_and_unescape_html(self, text):
+        text = strip_tags(text).strip()
+        return html.unescape(text)
```

Comment:
sorry for all those comments:
it would make sense to call `html.unescape` prior to`strip()` as it might replace `&nbsp;` with actual whitespaces which then may be removed by strip